src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java

```diff
@@ -43,13 +47,27 @@ public ResponseEntity<Void> processPcsStatement(@RequestHeader("x-request-id") S
         return ResponseEntity.status(HttpStatus.OK).build();
     }
 
+    @GetMapping("/company/{company_number}/persons-with-significant-control-statements")
+    public ResponseEntity<StatementList> searchPscStatementsList(
+            @PathVariable String company_number,
+            @RequestParam(value = "items_per_page", required = false) final Integer itemsPerPage,
+            @RequestParam(value = "start_index", required = false) final Integer startIndex) throws JsonProcessingException, ResourceNotFoundException {
+        logger.info(String.format("Retrieving psc statement list data for company number %s, start index %d, items %d", company_number,
+                Optional.ofNullable(startIndex).orElse(0),
+                Optional.ofNullable(itemsPerPage).orElse(25)));
+        StatementList statementList = pscStatementService.retrievePscStatementListFromDb(company_number,
+                Optional.ofNullable(startIndex).orElse(0),
+                Optional.ofNullable(itemsPerPage).orElse(25));
```

Comment:
We're calling Optional.ofNullable twice for each variable here. Is it worth these not being final and resetting the var value if null instead?